ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -166,6 +166,28 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: debug-winc-aws-ipi
+  optional: true
+  run_if_changed: test/extended/winc/
+  steps:
+    cluster_profile: aws-qe
+    dependency_overrides:
+      BUNDLE_IMAGE: quay.io/winc/wmco-bundle:latest
```

Comment:
Where this bundle image from? I didn't see any step for this bundle image building.

Reply:
If you want to run those e2e cases of https://github.com/openshift/openshift-tests-private, you don’t need to do that.